.gitignore

```diff
@@ -6,7 +6,7 @@ yarn-error.log*
 lerna-debug.log*
 .idea
 .origintrail_noderc
-.*_origintrail_noderc
+.*_origintrail_noderc.json
```

Comment:
Why we added .json to noderc file name? Is this only for testing?

Reply:
This is only for local network setup configuration files. In general, i think it's better to have file extensions defined, but we can't do this for the mainnet and testnet env configs at the moment as it would require a migration